pkgs/applications/networking/ipget/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, fetchgx }:
+
+buildGoPackage rec {
+  name = "ipget-${version}";
+  version = "0.2.5";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/ipfs/ipget";
+  
+  extraSrcPaths = [
+    (fetchgx {
+      inherit name src;
+      sha256 = "1d4w8zl5mcppn3d4bl7qdkiqlf8gi3z2a62nygx17bqpa3da8cf3";
+    })
+  ];
+ 
+  goDeps = ../../../tools/package-management/gx/deps.nix;
```

Comment:
Is it truly the same dependecies set? It looks like it would be better to ipget to have its own deps here. You can try to use go2nix to generate new deps.nix.

Reply:
I tried and I couldn't do even the step 1 of your instruction really:
```
% export GOPATH=`pwd`
% go get -d github.com/ipfs/ipget
package gx/ipfs/QmQa2wf1sLFKkjHCVEbna8y5qhdMjL8vtTJSAc48vZGTer/go-ipfs/path: unrecognized import path "gx/ipfs/QmQa2wf1sLFKkjHCVEbna8y5qhdMjL8vtTJSAc48vZGTer/go-ipfs/path" (import path does not begin with hostname)
package gx/ipfs/QmVcLF2CgjQb5BWmYFWsDfxDjbzBfcChfdHRedxeL3dV4K/cli: unrecognized import path "gx/ipfs/QmVcLF2CgjQb5BWmYFWsDfxDjbzBfcChfdHRedxeL3dV4K/cli" (import path does not begin with hostname)
package gx/ipfs/Qmdk8Ea9GkbwHr7UNKVYaLRwwHSt69xBXuSvRVyNWZ9sZE/fallback-ipfs-shell: unrecognized import path "gx/ipfs/Qmdk8Ea9GkbwHr7UNKVYaLRwwHSt69xBXuSvRVyNWZ9sZE/fallback-ipfs-shell" (import path does not begin with hostname)
```
You see, ipfs projects use their own dependency management stack called ```gx```, which, according to them, is a different thing from what is usually used in go. However it seems like not everything that is needed for build gets downloaded by ```gx```, this is why this needed goDeps attribute as well.

Reply:
I have a hunch that these missing dependencies are really just dependencies  of ```gx``` that are somehow leaking through into the ```ipget``` build.

Reply:
So `go2nix` would need to support `gx` to be used here.